doc/debugging.md

```diff
@@ -0,0 +1,69 @@
+# Debugging {#debugging}
+
+## Available tools
+
+xkbcommon provides multiple tools for debugging. Please consult the manual pages
+`man xkbcli` for the complete documentation of each tools.
+
+@note These tools may not be installed by default; please ensure you have the
+package `libxkbcommon-tools` installed.
```

Comment:
I *think* better english would be 
```suggestion
`libxkbcommon-tools` package installed.
```
note that in Fedora it's called `libxbkcommon-utils` so might be worth adding a handwave for the package name.